src/views/Home/AlertsTable/index.tsx

```diff
@@ -103,11 +105,15 @@ function AlertsTable() {
         filter: {},
     });
 
-    const variables = useMemo<{ filters: AlertFilter, pagination: OffsetPaginationInput }>(() => ({
+    // eslint-disable-next-line max-len
+    const variables = useMemo<{ filters: AlertFilter, pagination: OffsetPaginationInput, order: AlertOrder }>(() => ({
         pagination: {
             offset: page,
             limit,
         },
+        order: {
+            sent: (ordering === 'sent' ? 'ASC' : 'DESC') as Ordering,
+        },
```

Comment:
Maybe this is not the correct way to order. Let's use the sortState value to create ordering.